helm/sematic-server/values.yaml

```diff
@@ -69,19 +69,19 @@ deployment:
       path: /
       port: http
       scheme: HTTP
-    initialDelaySeconds: 10
+    initialDelaySeconds: 200
```

Comment:
instead of doing this, it'll be better to configure startup probes for the app.  startup probes run after the container is running, and they disable readiness/liveness checks until they succeed.  so the container starts, and startup probes run until they succeed, which then indicates to k8s that the app is up and running inside the container.  then it starts running readiness probes until they succeed, which indicates to k8s that the app is ready to accept traffic.  as a steady state check, liveness probes run to ensure that the app continues to be healthy and ready to accept traffic.

Reply:
I endorse this proposal.

Reply:
I actually thought that readinessProbe was essentially this! Added a startup probe, and it seems to work well.

Btw, I reverted the readiness probe to its original settings once I got the startup probe working. But there are still some useful stats in my comment [here](https://github.com/sematic-ai/sematic/pull/881/files#r1222329113), so I've linked it to keep them from getting buried.